src/AWS.Deploy.CLI/Commands/DeployCommand.cs

```diff
@@ -130,6 +130,23 @@ public async Task ExecuteAsync(string applicationName, string deploymentProjectP
 
             await CreateDeploymentBundle(orchestrator, selectedRecommendation, cloudApplication);
 
+            if (!string.IsNullOrEmpty(saveSettingsFilePath))
+            {
+                var success = true;
+                try
+                {
+                    await _deploymentSettingsHandler.SaveSettings(saveSettingsFilePath, selectedRecommendation, cloudApplication, _session);
+                }
+                catch (FailedToSaveDeploymentSettingsException ex)
+                {
+                    success = false;
+                    _toolInteractiveService.WriteErrorLine(ex.Message);
+                }
+
+                if (success)
+                    _toolInteractiveService.WriteLine($"{Environment.NewLine} Successfully saved the deployment settings at {saveSettingsFilePath}");

```

Comment:
You can add this `WriteLine` just after the `SaveSettings` call on line 138. If there was an exception then `WriteLine` wouldn't be called. Then you can simplify the code and get rid of the local `success` variable.